### ISSUE TITLE
Screensaver never centered / always in a different position (x-Axis) + exited too easily

### ISSUE THREAD

**Issue by Der-Flip** [qw123wh/Aospdeskclock#23]
I noticed that when entering the screen saver the clock face is (almost) never in the same position and (almost) never centered.

https://user-images.githubusercontent.com/18213041/182580147-6819ff01-4996-4e1d-9721-21ff4dece031.mp4

furthermore I think it's too easy to exit the screen saver mode. Either a confirmation or a "tap to hold" would make this screen a lot more helpful.

I would contribute myself, sadly I'm as useful as dead pixels on a screen when it comes to programming.

**Comment by qw123wh** [comment]
Yes I know it's pretty useless as it currently is.  I'll leave the problem open, maybe someone will be able to solve it
